### DISCUSSION TITLE
"Edit on Github" button gives invalid links (404)

### DISCUSSION THREAD

**Discussion by Khang-NT** [GitbookIO/community#600] [0. General bugs and questions] [answered]
Please check my pages: https://docs.zklite.io/
The button "Edit on Github" redirect to 404 page, for example: https://github.com/zklite-exchange/docs.git/README.md

**Comment by addisonschultz** [comment, marked as answer]
Very odd! Can you try something for me, and try to re-connect GitSync and see if that updates it correctly?

**Comment by Khang-NT** [comment]
Yeah, it works after re-connect
